server.js

```diff
@@ -2,29 +2,13 @@ const Hapi = require('hapi');
 const config = require('config');
 
 const routes = require('./routes');
-const plugins = require('./plugins');
-const logger = require('./server/utils/logger');
 
-const server = new Hapi.Server();
-
-server.connection({
+const server = Hapi.Server({
 	port: config.get('app.port'),
 });
 
 // attach routes here
 server.route(routes);
 
-// register plugins
-const registerPlugins = async () => {
```

Comment:
Plugin registration code is already in `async-await` pattern. Is there any reason to remove this?

Reply:
@Arjun-sna It is not working due to the async await pattern which it is failing. It is mainly due to the plugin.js file where how the plugins are declared. We cannot use the older structure for `plugins` array in plugin.js. I have raised a separate issue #18.

Reply:
Ok. then we can merge these changes now. so people working on the plugin issue will have these changes.

Reply:
Will discuss this changes when we meet on Monday.

Reply:
@rahulthampi We can actually merge this since we can add plugins later I guess. Discussing on Monday is fine, but this is actually required if we want to do anything in terms of writing an API.

Reply:
done already!